apps/web/src/app/(dashboard)/documents/empty-state.tsx

```diff
@@ -32,7 +32,7 @@ export const EmptyDocumentState = ({ status }: EmptyDocumentProps) => {
     .otherwise(() => ({
       title: 'Nothing to do',
       message:
-        'All documents are currently actioned. Any new documents are sent or recieved they will start to appear here.',
+        'All documents have been processed. Any new documents that are sent or received will start to appear here.',
```

Comment:
Can this be changed to `Any new documents that are sent or received will show here.',`, please?

Reply:
Should i replace the whole sentence? or just the `Any new document...` part?

Reply:
@Bitnagar only the second one - `Any new documents...` please.

Reply:
Done. Let me know if any other change is there.